projects/plugins/boost/app/assets/src/js/elements/TimeAgo.svelte

```diff
@@ -19,6 +19,8 @@
 	} );
 </script>
 
-<span title={time.toLocaleString()} class="time-ago">
-	{$label}
-</span>
+{#if ! isNaN( time.getTime() )}
```

Comment:
If time is a date can getTime ever return `NaN`?

Reply:
Yes
```
isNaN(new Date( NaN )) && isNaN(new Date( undefined ))
```

will return true